xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/RendererOMX.cpp

```diff
@@ -27,6 +27,7 @@
 #include "utils/GLUtils.h"
 #include "settings/MediaSettings.h"
 #include "windowing/WindowingFactory.h"
+#include "DVDCodecs/Video/DVDVideoCodecIMX.h"
```

Comment:
If I use `#include "DVDCodecs/Video/DVDVideoCodecOpenMax.h"` instead i get the error below. Could this be related to my hardware? I'm compiling on a cubox-i.

```
$ make
CPP     xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/RendererOMX.o
RendererOMX.cpp: In member function 'virtual void CRendererOMX::AddVideoPictureHW(DVDVideoPicture&, int)':
RendererOMX.cpp:71:22: error: invalid use of incomplete type 'class CDVDVideoCodecIMXBuffer'
     picture.IMXBuffer->Lock();
                      ^
In file included from /home/builder/xbmc-imx-git/src/xbmc/xbmc/cores/VideoPlayer/VideoRenderers/LinuxRendererGLES.h:35:0,
                 from RendererOMX.h:27,
                 from RendererOMX.cpp:21:
/home/builder/xbmc-imx-git/src/xbmc/xbmc/cores/VideoPlayer/DVDCodecs/Video/DVDVideoCodec.h:55:7: note: forward declaration of 'class CDVDVideoCodecIMXBuffer'
 class CDVDVideoCodecIMXBuffer;
       ^
../../../../../Makefile.include:97: recipe for target 'RendererOMX.o' failed
make: *** [RendererOMX.o] Error 1
```